src/common/slash.ts

```diff
@@ -36,6 +38,7 @@ export interface ISlashCommand {
   name: string;
   description: string;
   options?: ApplicationCommandOptionData[];
+  minPermissions?: PermissionResolvable;
```

Comment:
We can probably use the same discord name for it as it's a bit more accurate.
```suggestion
  defaultMemberPermissions?: PermissionResolvable;
```